onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -793,19 +797,16 @@ of [N, 0] then [N, 0].
           "An optional string. Token's regular expression in basic POSIX format"
           " (http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap09.html#tag_09_03)."
           " If set, tokenizer may produce tokens matching the specified pattern. Note that one and only of"
-          " 'tokenexp' and 'separators' should be set.",
+          " 'tokenexp' and 'sepexp' should be set.",
```

Comment:
>sepexp [](start = 28, length = 6)

separators

Reply:
sepexp -> separators

Reply:
Humm. I like `sepexps` or `separator_expressions` (if ONNX attributes should have full names) a bit more.

Reply:
So, what about the compatibility? Do we know if there are current users for the op?

---
In reply to: [276865069](https://github.com/Microsoft/onnxruntime/pull/857#discussion_r276865069) [](ancestors = 276865069)

Reply:
I agree with Xavier that we want to preserve current code and keep it working. It depends on the name 'separators'

---
In reply to: [277052645](https://github.com/Microsoft/onnxruntime/pull/857#discussion_r277052645) [](ancestors = 277052645,276865069)